### DISCUSSION TITLE
attu approxEntityCount value didn't change when deleting successfully

### DISCUSSION THREAD

**Discussion by stoneLee81** [milvus-io/milvus#26650] [Q&A and General discussion]
In the attu interface, the deleted data, the data is indeed deleted, but the approxEntityCount has not changed.
<img width="466" alt="image" src="https://github.com/milvus-io/milvus/assets/18001708/d7e9f07c-c38a-4e02-8896-1f7e92bc7efe">
in fact, there is no data left, i deleted all, but the approxEntityCount field is still show 3 data left.

**Comment by yhmo** [comment]
Deleted entities are not counted in the approxEntityCount, this is a tech limitation for current versions. In the v2.3, we will provide a new api to count accurate number of entities. approxEntityCount is fast but not accurate, the new api is accurate but is slower than approxEntityCount.
